### DISCUSSION TITLE
RS485 on the WT32-ETH01

### DISCUSSION THREAD

**Discussion by bobwho69** [ldijkman/WT32-ETH01-LAN-8720-RJ45-#15] [General]
Hi there, I am wanting to get the RS485 working natively on this board as I am struggling to troubleshoot my problems with another esp32 and addon module (suspect its the isolation protection)

I cant find any info on how the 485_en pin functions and how to set a serial port for rs485 comms

Has anybody tried to get this feature working?

**Comment by AlexanderRose82** [comment]
Use a TTL-RS485 converter: https://www.printables.com/model/770951-rs485-esp32-wt32-eth01-usb-type-c-snap-fit-case
